components/create_comment/create_comment.jsx

```diff
@@ -704,8 +704,8 @@ class CreateComment extends React.PureComponent {
             if (index !== -1) {
                 uploadsInProgress.splice(index, 1);
 
-                if (this.refs.fileUpload && this.refs.fileUpload.getWrappedInstance() && this.refs.fileUpload.getWrappedInstance().getWrappedInstance()) {
-                    this.refs.fileUpload.getWrappedInstance().getWrappedInstance().cancelUpload(id);
+                if (this.refs.fileUpload && this.refs.fileUpload.getWrappedInstance()) {
```

Comment:
do you need `getWrappedInstance` even by using `forwardRef` ? that seems strange

Reply:
The remaining `getWrappedInstance` is actually from `connect`. If that gets updated to use `React.forwardRef`, then we can remove the second `getWrappedInstance`